apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx

```diff
@@ -0,0 +1,301 @@
+  padding: 0;
+  text-decoration: underline;
+  display: text;
+  cursor: pointer;
+  background: none;
+  color: white;
+  margin: 10px 0;
+`
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5em;
+  gap:10px;
+
+  p {
+    padding: 0 1em;
+  }
+`
+
+const ContentWrapper = styled.div`
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  padding: 1em;
+  text-align: center;
+`
+
+const Input = styled.input`
+  border: 1px solid var(${UI.COLOR_BORDER});
+  background: var(${UI.COLOR_PAPER_DARKER});
+  color: inherit;
+  width: 100%;
+  margin: 10px 0;
+  margin-top: 0;
+  padding: 10px;
+  border-radius: 12px;
+  outline: none;
+  font-size: 15px;
+  font-weight: bold;
+
+  &:focus {
+    border: 1px solid var(${UI.COLOR_PRIMARY});
+  }
+`
+
+const Row = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  margin: 10px;
+  width: 100%;
+
+  label {
+    margin: 10px;
+    flex-grow: 0;
+    width: 5em;
+  }
+
+  input,
+  textarea {
+    flex-grow: 1;
+  }
+
+  /* Chrome, Safari, Edge, Opera */
+    input::-webkit-outer-spin-button,
+    input::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+    }
+`
+
+const Dropdown = styled.div`
+  position: relative;
+  display: inline-block;
+  width:100%;
+`
+
+const DropdownContent = styled.div`
+  display: block;
+  background-color: var(${UI.COLOR_PAPER});
+  min-width: 80%;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  padding: 12px 16px;
+  bottom: 100%;
+  right: 0;
+  height: min-content;
+  z-index: 999;
+  width: 100%;
+  border-radius: 16px;
+  margin: auto auto 12px;
+
+  ${Media.upToLarge()} {
+    left: 0;
+    right: initial;
+  }
+
+  ${Media.upToMedium()} {
+    width: 100%;
+    position: fixed;
+    bottom: 56px;
+    margin: 0;
+    padding: 18px 24px;
+    box-shadow: 0 -100vh 0 100vh rgb(0 0 0 / 40%);
+    border-bottom: 1px solid var(${UI.COLOR_BORDER});
+    border-radius: 16px 16px 0 0;
+  }
+`
+
+const DropdownButton = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  border-radius: 12px;
+  background-color: var(${UI.COLOR_PAPER_DARKER});
+  padding:12px;
+  width:100%;
+
+  &:hover {
+    background-color: var(${UI.COLOR_PRIMARY_LIGHTER});
+    color:#000000;
+    /* > svg {
+      transform: rotate(-90deg);
+    } */
+  }
+
+  > svg {
+    --size: 12px;
+    color: inherit;
+    width: var(--size);
+    height: var(--size);
+    transition: transform 0.2s ease-in-out;
+  }
+`
+
+const SelectButton = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    cursor: pointer;
+    border-radius: 16px;
+    background-color: #18193B;
+    padding:1rem;
+    width:100%;
+
+    &:hover {
+    background-color: #65D9FF;
+    color:#000000;
```

Comment:
How do they handle the colors of the other components? Probably there is some kind of variable to make it easier to update, right? Can you check this?